frameworks/helloworld/tests/test_multiple_ports.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import logging
+from tests import config
+
+import sdk_install
+import sdk_plan
+import sdk_tasks
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+def test_install():
```

Comment:
Let's give this test a more meaningful name. It will make things clearer if we ever add additional tests.